extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -143,6 +143,7 @@ export interface SavedInfo {
 	closedPage: number;
 	databaseAssessment: string[];
 	databaseList: string[];
+	databaseInfoList: Map<string, SourceDatabaseInfo | undefined>;
```

Comment:
since you added this `databaseInfoList` in `SavedInfo`, looks like you need to add that here as well:

https://github.com/microsoft/azuredatastudio/blob/7a736b76fa367c3a83c9ee90883373d8da9312ef/extensions/sql-migration/src/dialog/retryMigration/retryMigrationDialog.ts#L37

currently this is causing the build to fail:

![image](https://user-images.githubusercontent.com/11844501/211080550-0afcdd3f-3678-4227-bd2e-0e14ffab3a26.png)